src/dotnet-svcutil/lib/src/Shared/Options/UpdateOptions.cs

```diff
@@ -97,6 +97,16 @@ public void MakePathsRelativeTo(DirectoryInfo optionsFileDirectory)
                     this.Inputs[idx] = new Uri(relPath, UriKind.Relative);
                 }
             }
+
```

Comment:
Only binary dependency type is affected, reconstruct the reference with reference.AssemblyName writes name instead of path to the .json file.